app/src/main/java/app/musikus/ui/theme/Theme.kt

```diff
@@ -74,47 +73,37 @@ fun MusikusTheme(
         }
     }
 
-    // source: https://gist.github.com/Khazbs/1f1f1b5c05f45dbfa465f249b1e20506
-    // and https://stackoverflow.com/questions/65610216/how-to-change-statusbar-color-in-jetpack-compose
-    val view = LocalView.current
-    if (!view.isInEditMode) {
```

Comment:
I did not notice any odd behavior while in edit mode (Sessions, Goals), is there a specific reason this has only been done in non-Edit mode?

Reply:
The "!view.isInEditMode" was copied from the above sources. We can remove it and see if anything breaks